docs/crd.md

```diff
@@ -88,6 +88,9 @@ FlinkCluster
             |__ secretName
             |__ keyFile
             |__ mountPath
+    |__ logConfig
+        |__ "log4j-console.properties"
```

Comment:
The type of field `logConfig` is `map[string]string`, `"log4j-console.properties"` and `"logback-console.xml"` are possible keys of the map. I think we should not add keys here, instead only document them in the field description below.